lib/getEmbeds.ts

```
@@ -24,70 +24,109 @@ export function isVideo(url?: string) {
+      casts.push(embed.cast_id);
     } else if (embed?.url) {
       if (isImg(embed.url)) {
         imgs.push(embed);
       } else if (isVideo(embed.url)) {
-        videos.push({
-          url: embed.url,
-        });
-      } else {
-        webpages.push({
+        videos.push(embed);
+      } else if (!!embed?.metadata?.html?.ogSiteName) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)